pkgs/development/tools/devtodo1/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, readline, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "devtodo-${version}";
```

Comment:
This name is inconsistent with the name of the derivation. I'd suggest renaming the derivation to `devtodo`, because upstream calls the packages `devtodo` and `devtodo2` and the tools `todo` and `todo2`.